mk/docs.mk

```diff
@@ -287,12 +287,12 @@ doc/book/index.html: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/trpl/*.md) | doc/
 	$(Q)rm -rf doc/book
 	$(Q)$(RUSTBOOK) build $(S)src/doc/trpl doc/book
 
-tarpl: doc/adv-book/index.html
+nomicon: doc/nomicon/index.html
 
-doc/adv-book/index.html: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/tarpl/*.md) | doc/
+doc/nomicon/index.html: $(RUSTBOOK_EXE) $(wildcard $(S)/src/doc/nomicon/*.md) | doc/
 	@$(call E, rustbook: $@)
-	$(Q)rm -rf doc/adv-book
-	$(Q)$(RUSTBOOK) build $(S)src/doc/tarpl doc/adv-book
+	$(Q)rm -rf doc/nomicon
+	$(Q)$(RUSTBOOK) build $(S)src/doc/nomicon doc/nomicon
```

Comment:
This translates to the ultimate URL of the doc and I wonder if 'doc/rustonomicon' would be clearer, more guessable.